server/services/core/config.service.ts

```diff
@@ -80,6 +80,7 @@ class ConfigService extends TcService {
       tianji: config.tianji,
       uploadFileLimit: config.storage.limit,
       emailVerification: config.emailVerification,
+      disableMsgpack: config.feature.disableMsgpack,
```

Comment:
ok here is another problem is name problem.
did you tested pass? because i think you tell frontend a field is `disableMsgpack` but using `disableSocketMsgpack`. i think its will not work. can you unify those name? thx

Reply:
I'll fix that later(